projects/packages/forms/src/dashboard/inbox/list.js

```diff
@@ -13,6 +14,7 @@ const COLUMNS = [
 	{
 		key: 'date',
 		label: __( 'Date', 'jetpack-forms' ),
+		getValue: item => dateI18n( 'M j, Y', item.date ),
```

Comment:
Instead of adding complexity to the `<Table>` component, if it's just a simple property, not a component or anything, couldn't we just add the new property/value to the item [here](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/src/dashboard/inbox/list.js#L45)?

Reply:
Yeah, adding the property value to the `tableItems` would do the job.
But I'd argue if we're indeed adding complexity to the `<Table>` component.

IMHO, I think the `getValue` property gives us the flexibility to compose the cell values the way we desire.
We could even get rid of the `component` and `getProps` properties, since we could use the `getValue` function to return any markup, like:

```Javascript
		getValue: ( item ) => (
			<Button href={ item.entry_permalink } variant="link">{ item.source }</Button>
		),
```

Also, it'd make it clear to understand what a column value represents, since it'd be tied to the `COLUMNS` instead of anywhere else in the code.

Reply:
Hmm... fair enough, maybe that's indeed simple. I'll allow it 😉.
But then maybe let's remove `component` and `getProps` later like you suggested - doesn't need to be in this PR though :)

Reply:
Cool! Thanks for your blessing! 😆 
I've already removed the `component` and `getProps` and made the necessary adjustments to the code to work with the `getValue` function

Reply:
Thanks for cleaning it up! 🙇‍♂️